docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+
+namespace Samples.Dynamic
+{
+    public static class CalculateFeatureContribution
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+
+            // Create a small dataset.
+            var samples = Data.GenerateData();
+
+            // Convert training data to IDataView.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // Create a pipeline to concatenate the features into a feature vector and normalize it.
```

Comment:
This doesn't look a part of FCC. May we start with raw data?

Reply:
I wanted to show that FCC works on featurized data, not on the original columns, so I wanted to make it explicit.